### PR TITLE
Update bookings-in-outlook.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the PS cmdlet as works fine with "Anonymous:CalendarSharingFreeBusySimple" sharing policy

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9430

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9453

**Comment by tiburd** [comment]
@get-itips @dario-ms @tseward 
Hi! Can you please review the changes to the PowerShell blocks in this PR?
Thanks!

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 1]
```diff
@@ -57,20 +57,20 @@ For more information, see the [Bookings with me Microsoft 365 Roadmap item](http
...
+   To turn on Bookings with me without access to Bookings, block access to Microsoft Bookings using the [OWA Mailbox policy PowerShell command](/powershell/module/exchange/set-owamailboxpolicy) or follow the instructions here: [Turn Microsoft Bookings on or off](turn-bookings-on-or-off.md).
 
 2. Calendar FreeBusy Anonymous sharing must be enabled to use Bookings with me. This allows the Bookings page to have access to the free/busy information in your Outlook calendar. Use PowerShell to check the status.
 
    ```PowerShell
      Get-SharingPolicy -Identity "Default Sharing Policy" | fl Domains 
    ```
 
-    "Anonymous:CalendarSharingFreeBusyReviewer"" should be one of the domains in the response.
+    "Anonymous:CalendarSharingFreeBusySimple"" should be one of the domains in the response.
```
Comment (ghost) [inline microsoft-365/bookings/bookings-in-outlook.md]: Contains an extra closing quote.

**Review by ghost — CHANGES_REQUESTED** [review]

File: microsoft-365/bookings/bookings-in-outlook.md (hunk 1 above)
Comment (msbemba) [reply microsoft-365/bookings/bookings-in-outlook.md]: Thank you. this has been fixed

**Review by dariomws — COMMENTED** [review]
Hi @msbemba,

I don't understand this PR because you're proposing something different here:
https://github.com/MicrosoftDocs/microsoft-365-docs/pull/9270

Could you explain it in more detail?

**Comment by dariomws** [comment]
This is the setting in my demo tenant:

Domains : {Anonymous:CalendarSharingFreeBusyReviewer, *:CalendarSharingFreeBusySimple}

Which I understand is the default setting for O365 tenants. CalendarSharingFreeBusySimple is enough for Bookings with me? Did you test it? I think it should work but we need to confirm this before applying an update to this article.
If we confirm it, I think we should explain that the default setting is CalendarSharingFreeBusyReviewer but CalendarSharingFreeBusySimple is enough.

@kwekuako, maybe you can also provide your feedback here as I see that you added this information to the article.

These are my thoughts.

Thanks!

**Comment by msbemba** [comment]
@dariomws Thank you for checking this. I opened up a ticket and requested engineers to confirm with the booking team. Looks like any of the below permissions will work and it works during testing phase.

    "Anonymous:CalendarSharingFreeBusySimple": Share free/busy hours only.
    "Anonymous:CalendarSharingFreeBusyDetail": Share free/busy hours, subject and location.
    "Anonymous:CalendarSharingFreeBusyReviewer": Share free/busy hours, subject, location and the body of the message or calendar item.
    "Anonymous:ContactsSharing": Share contacts only.

Anonymous:CalendarSharingFreeBusySimple is enough however, the other options can give more options to the enduser.

Thanks
Sri

**Comment by get-itips** [comment]
@msbemba Should this PR https://github.com/MicrosoftDocs/microsoft-365-docs/pull/9270 be replaced by this one? which one should we review? Thanks

**Review by get-itips — COMMENTED** [review]
@msbemba  Should this PR https://github.com/MicrosoftDocs/microsoft-365-docs/pull/9270 be replaced by this one? Which one should we review? Thanks

**Comment by get-itips** [comment]
@msbemba The default appears to be what @dariomws has (I checked in different tenants)

So I don't think we should instruct people to change the default (from Anonymous:CalendarSharingFreeBusyReviewer to CalendarSharingFreeBusySimple) **as we do not know the impact in other applications and services, outside of Bookings**.

I'll try to test this today however I think the best would be to explain that, this is the expected default and that you could change it to Simple but that could impact other services that use this setting.

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 2]
```diff
@@ -57,20 +57,20 @@ For more information, see the [Bookings with me Microsoft 365 Roadmap item](http
...
+   To turn on Bookings with me without access to Bookings, block access to Microsoft Bookings using the [OWA Mailbox policy PowerShell command](/powershell/module/exchange/set-owamailboxpolicy) or follow the instructions here: [Turn Microsoft Bookings on or off](turn-bookings-on-or-off.md).
 
 2. Calendar FreeBusy Anonymous sharing must be enabled to use Bookings with me. This allows the Bookings page to have access to the free/busy information in your Outlook calendar. Use PowerShell to check the status.
 
    ```PowerShell
      Get-SharingPolicy -Identity "Default Sharing Policy" | fl Domains 
    ```
 
-    "Anonymous:CalendarSharingFreeBusyReviewer"" should be one of the domains in the response.
+    "Anonymous:CalendarSharingFreeBusySimple" should be one of the domains in the response.
```
Comment (dariomws) [inline microsoft-365/bookings/bookings-in-outlook.md]:
```suggestion
    "Anonymous:<SharingPolicyAction>" must be one of the domains in the response. <SharingPolicyAction> value can be CalendarSharingFreeBusySimple, CalendarSharingFreeBusyDetail, or  CalendarSharingFreeBusyReviewer (default).
```

**Review by dariomws — CHANGES_REQUESTED** [review]
Maybe this could be enough...

What do you think @get-itips?

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 3]
```diff
@@ -57,20 +57,20 @@ For more information, see the [Bookings with me Microsoft 365 Roadmap item](http
...
 
    To enable anonymous sharing, use the following command.
 
    ```PowerShell
-     Set-SharingPolicy "Default Sharing Policy" -Domains @{Add="Anonymous:CalendarSharingFreeBusyReviewer"}
+     Set-SharingPolicy "Default Sharing Policy" -Domains @{Add="Anonymous:CalendarSharingFreeBusySimple"}
    ```
```
Comment (dariomws) [inline microsoft-365/bookings/bookings-in-outlook.md:74]:
```suggestion
     Set-SharingPolicy "Default Sharing Policy" -Domains @{Add="Anonymous:CalendarSharingFreeBusySimple"}
   ```

For more information, see [Set-SharingPolicy](/powershell/module/exchange/set-sharingpolicy).
```

**Review by dariomws — CHANGES_REQUESTED** [review]
Minor update

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 4]
```diff
@@ -57,20 +57,20 @@ For more information, see the [Bookings with me Microsoft 365 Roadmap item](http
...
```
Comment (dariomws) [inline microsoft-365/bookings/bookings-in-outlook.md:77]: For more information, see [Set-SharingPolicy](/powershell/module/exchange/set-sharingpolicy).

**Review by dariomws — CHANGES_REQUESTED** [review]

File: microsoft-365/bookings/bookings-in-outlook.md (hunk 4 above)
Comment (msbemba) [reply microsoft-365/bookings/bookings-in-outlook.md:77]: Made the changes. Thank you.

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 5]
```diff
@@ -57,21 +57,23 @@ For more information, see the [Bookings with me Microsoft 365 Roadmap item](http
...
+    "Anonymous:<SharingPolicyAction>" must be one of the domains in the response. <SharingPolicyAction> value can be CalendarSharingFreeBusySimple, CalendarSharingFreeBusyDetail, or  CalendarSharingFreeBusyReviewer (default).
 
    To enable anonymous sharing, use the following command.
 
    ```PowerShell
-     Set-SharingPolicy "Default Sharing Policy" -Domains @{Add="Anonymous:CalendarSharingFreeBusyReviewer"}
+     Set-SharingPolicy "Default Sharing Policy" -Domains @{Add="Anonymous:CalendarSharingFreeBusySimple"}
    ```
+  
+  For more information, see [Set-SharingPolicy](/powershell/module/exchange/set-sharingpolicy)
```
Comment (dariomws) [inline microsoft-365/bookings/bookings-in-outlook.md]:
```suggestion
  For more information, see [Set-SharingPolicy](/powershell/module/exchange/set-sharingpolicy).
```

**Review by dariomws — APPROVED** [review]
I approve considering that you opened a support ticket to confirm that CalendarSharingFreeBusySimple is enough.

Could you kindly share the ticket number?

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 6]
```diff
@@ -57,21 +57,23 @@ For more information, see the [Bookings with me Microsoft 365 Roadmap item](http
...
+   To turn on Bookings with me without access to Bookings, block access to Microsoft Bookings using the [OWA Mailbox policy PowerShell command](/powershell/module/exchange/set-owamailboxpolicy) or follow the instructions here: [Turn Microsoft Bookings on or off](turn-bookings-on-or-off.md).
 
 2. Calendar FreeBusy Anonymous sharing must be enabled to use Bookings with me. This allows the Bookings page to have access to the free/busy information in your Outlook calendar. Use PowerShell to check the status.
 
    ```PowerShell
      Get-SharingPolicy -Identity "Default Sharing Policy" | fl Domains 
    ```
 
-    "Anonymous:CalendarSharingFreeBusyReviewer"" should be one of the domains in the response.
+    "Anonymous:<SharingPolicyAction>" must be one of the domains in the response. <SharingPolicyAction> value can be CalendarSharingFreeBusySimple, CalendarSharingFreeBusyDetail, or  CalendarSharingFreeBusyReviewer (default).
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-in-outlook.md]:
```suggestion
    "Anonymous:<SharingPolicyAction>" must be one of the domains in the response. <SharingPolicyAction> value can be CalendarSharingFreeBusySimple, CalendarSharingFreeBusyDetail, or CalendarSharingFreeBusyReviewer (default).
```

**Comment by get-itips** [comment]
It seems that the Bookings with me will not share any other detail in the UX anyway (subject, location and the body of the message or calendar item), just fyi if we need to clarify that in the future.

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 7]
```diff
@@ -57,21 +57,23 @@ For more information, see the [Bookings with me Microsoft 365 Roadmap item](http
...
 
    To enable anonymous sharing, use the following command.
 
    ```PowerShell
-     Set-SharingPolicy "Default Sharing Policy" -Domains @{Add="Anonymous:CalendarSharingFreeBusyReviewer"}
+     Set-SharingPolicy "Default Sharing Policy" -Domains @{Add="Anonymous:CalendarSharingFreeBusySimple"}
```
Comment (ekaplanb) [inline microsoft-365/bookings/bookings-in-outlook.md:73]: Why not using the default of new teants: "CalendarFreeBusyReviewer"

Comment (dariomws) [reply microsoft-365/bookings/bookings-in-outlook.md:73]: Due to the least privilege principle. If anonymous sharing is disabled, this is the minimum requirement to enable it.
Both issue raisers are complaining about allowing additional permissions to anonymous.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by msbemba** [comment]
@yogkumgit We are waiting for @ekaplanb  response to @dariomws

File: microsoft-365/bookings/bookings-in-outlook.md (hunk 7 above)
Comment (msbemba) [reply microsoft-365/bookings/bookings-in-outlook.md:73]: Thank you @dariomws  and @ekaplanb

**Comment by get-itips** [comment]
@msbemba Can you take a look at the build warnings? Thanks

**Review by ghost — APPROVED** [review]

**Pull request merged**